client/js/models/archive.js

```diff
@@ -79,7 +80,6 @@ module.exports = {
   effects: {
     new: function (data, state, send, done) {
       drive = getDrive()
-      var _fs = {}
```

Comment:
@karissa i just committed some changes to the archive model that is using the pattern that @clkao is talking about: you'll need to use `var` hoisting, see the following for example pattern: https://github.com/datproject/dat.land/blob/master/client/js/models/archive.js#L9